ska.shell/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: ska.shell  # lower case name of package, may contain '-' but no spaces
+  version: {{ data.get('version') }}
```

Comment:
Oh wait, I probably just need to fix the templating to work with Py3 due to unicode issues.